### PR TITLE
Improve wear open loop notification

### PR DESCRIPTION
**Pull request by olorinmaia**
My proposal to fix/improve https://github.com/nightscout/AndroidAPS/issues/3631

Before:
image

After:
image
image

### CODE REVIEW

File: core/objects/src/main/kotlin/app/aaps/core/objects/aps/DetermineBasalResult.kt [hunk 1]
```diff
@@ -124,9 +124,9 @@ class DetermineBasalResult @Inject constructor(val injector: HasAndroidInjector)
             // rate
             var ret: String = if (rate == 0.0 && duration == 0) "${rh.gs(R.string.cancel_temp)} "
             else if (rate == -1.0) "${rh.gs(R.string.let_temp_basal_run)}\n"
-            else if (usePercent) "${rh.gs(R.string.rate)}: ${decimalFormatter.to2Decimal(percent.toDouble())}% (${decimalFormatter.to2Decimal(percent * pump.baseBasalRate / 100.0)} U/h) " +
+            else if (usePercent) "${rh.gs(R.string.rate)}: ${decimalFormatter.to2Decimal(percent.toDouble())}% (${decimalFormatter.to2Decimal(percent * pump.baseBasalRate / 100.0)} ${rh.gs(R.string.profile_ins_units_per_hour)}) " +
```
Comment (MilosKozak) [inline core/objects/src/main/kotlin/app/aaps/core/objects/aps/DetermineBasalResult.kt:127]: this is historicaly wrong. it's not possible to translate it correctly to languages that differs too much.
create new string like "Rate: %1$.2f%% (%2$.2f U/h)"
I believe many of such strings already exists can be reused

Comment (olorinmaia) [reply core/objects/src/main/kotlin/app/aaps/core/objects/aps/DetermineBasalResult.kt]: For some reason turning this into a single rh.gs proved to be difficult. I will look at it later.

**Comment by MilosKozak** [comment]
@Philoul can you verify?

**Comment by Philoul** [comment]
I opened the loop a few min to test the PR 😉...
Looks good, same notification on watch compared to phone, and I could validate the proposal from the watch to apply proposed action...

**Pull request merged**